### PR TITLE
chore: announce patches

### PR DESCRIPTION
**Pull request by caarlos0**
announce releases 

refs https://github.com/charmbracelet/meta/pull/63

### CODE REVIEW

**Comment by caarlos0** [comment]
@aymanbagabas do you know if we need to map secrets like we do with goreleaser token et al? 

if so, I need to change some things around in https://github.com/charmbracelet/meta/pull/63

**Comment by aymanbagabas** [comment]
> @aymanbagabas do you know if we need to map secrets like we do with goreleaser token et al?
> 
> if so, I need to change some things around in [charmbracelet/meta#63](https://github.com/charmbracelet/meta/pull/63)

Ugh, we need to map the secrets in the goreleaser workflow in every repository. The `secrets` property in meta refers to the secret inputs *NOT* the global secret store because the workflow in meta is a `workflow_call` :/

**Comment by caarlos0** [comment]
ah, figures, ight, will do that

**Comment by caarlos0** [comment]
https://github.com/charmbracelet/meta/pull/64

File: .github/workflows/goreleaser.yml [hunk 1]
```diff
@@ -17,3 +17,14 @@ jobs:
...
       goreleaser_key: ${{ secrets.GORELEASER_KEY }}
+      twitter_consumer_key: ${{ secrets.twitter_consumer_key }}
+      twitter_consumer_secret: ${{ secrets.twitter_consumer_secret }}
+      twitter_access_token: ${{ secrets.twitter_access_token }}
+      twitter_access_token_secret: ${{ secrets.twitter_access_token_secret }}
+      mastodon_client_id: ${{ secrets.mastodon_client_id }}
+      mastodon_client_secret: ${{ secrets.mastodon_client_secret }}
+      mastodon_access_token: ${{ secrets.mastodon_access_token }}
+      discord_webhook_id: ${{ secrets.discord_webhook_id }}
+      discord_webhook_token: ${{ secrets.discord_webhook_token }}
```
Comment (aymanbagabas) [inline .github/workflows/goreleaser.yml:28]: Uppercase

**Review by aymanbagabas — CHANGES_REQUESTED** [review]

**Pull request merged**